staking/programs/staking/src/state/vesting.rs

```diff
@@ -709,6 +714,28 @@ pub mod tests {
             }
         }
 
+        for timestamp in 0..(START_TIMESTAMP + (PERIOD_DURATION * NUM_PERIODS + 1) as i64) {
```

Comment:
You could change the increment of this for loop to be `PERIOD_DURATION` and then it would run fast for big values of `PERIOD_DURATION`